src/main/java/meditracker/medication/Medication.java

```diff
@@ -42,20 +55,44 @@ public void setName(String name) {
         this.name = name;
     }
 
-    public String getQuantity() {
+    public Double getQuantity() {
         return quantity;
     }
 
     public void setQuantity(String quantity) {
-        this.quantity = quantity;
+        this.quantity = Double.parseDouble(quantity);
     }
 
-    public String getDosage() {
+    public Double getDosage() {
         return dosage;
     }
 
     public void setDosage(String dosage) {
```

Comment:
Same as above, might be better to standardise the interface and just take in `Double` instead of `String` (so you don't have to handle the parsing error here also)

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/75/commits/cbf6980cd51719e0d279630c0d0e2c04650925cd